keep.tex

```diff
@@ -518,7 +519,7 @@ \subsection{Concerns with active attacks}
 token native to the system. Significant disruption of the network
 should lead to a drop in the value of the token, incentivizing
 provider honesty, lest they devalue their holdings. This scheme also
-provides resistance to Sybil attacks- an active attacker would need to
+provides resistance to Sybil attacks--- an active attacker would need to
```

Comment:
Could you link to discussions of Sybil attacks on Ethereum as Keep will be sitting on that chain initially? Any value in explicitly calling out how Keep is resistant to Sybil attacks on the chain it sits on (ie. no transitive property between a weakness in Ethereum and Keep as Keep has it's own incentive and lockup system)?

Reply:
Definitely worthwhile, though out of scope for this PR- tracking it in https://github.com/keep-network/whitepaper/issues/19